app/metrics.yaml

```diff
@@ -1856,3 +1856,17 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+startup_timeline_telemetry:
+  gecko_runtime_create:
+    type: timespan
+    description: >
```

Comment:
As currently defined, this would be sent in the `metrics` ping only _once_. The metric will be cleared after the ping is collected and won't be available in other `metrics` pings that may be generated within this session.

If that's the intent, then nothing to do. If you want this metric to be available in most of the `metrics` pings being generated, then you need to add `lifetime: application` here. [See our docs](https://mozilla.github.io/glean/book/user/metric-parameters.html#optional-metric-parameters).

If you send this within a custom ping, then leaving the default lifetime may be fine.